app.py

```diff
@@ -1,17 +1,34 @@
 from flask import Flask
 from flask import request
+from nltk.util import pr
 from textblob import TextBlob
 from flask_cors import CORS, cross_origin
+import enchant
+import sys
 
 app = Flask(__name__)
 CORS(app)
-@app.route('/', methods = ['GET','POST'])
-def index():
+irish_dict = enchant.PyPWL("words2.txt")
+
+@app.route('/en', methods = ['GET','POST'])
+def english():
     data = str(request.form['words'])
     data1 = TextBlob(data)
+    print(data1)
     return str(data1.correct())
 
+@app.route('/ir', methods = ['GET','POST'])
+def irish():
+    res =""
+    irish_words = (request.form['words']).split(" ")
+    for irish_word in irish_words:
```

Comment:
When executed and after typing a word it is displaying two results

Reply:
OK I will correct the issue